dockyard/common/cluster/cluster.py

```diff
@@ -0,0 +1,14 @@
+from oslo_config import cfg
+
+class Cluster(object):
+    def __init__(self):
+        pass
+
+    def register(self, cluster_id, host_ip, port):
+        pass
+
+    def unregister(self, cluster_id, host_ip):
+        pass 
+
+    def get_hosts(self):
+        return cfg.CONF.membership.hosts
```

Comment:
Write a better membership algorithm because this one always adds its own host only.

We have to decide about membership protocol to be used in this product.